projects/addon-mobile/components/mobile-calendar/mobile-calendar.component.ts

```diff
@@ -139,17 +136,34 @@ export class TuiMobileCalendarComponent {
     }
 
     ngAfterViewInit() {
-        // Virtual scroll has not yet rendered items even in ngAfterViewInit
```

Comment:
Let's leave this comment. It explains why we wait for `take(1)` of `scrolledIndexChange`.